foundation_cms/static/js/blocks/impact_numbers_block.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Minor improvement suggestion -

Let's extract the hard-coded selector strings into a single `SELECTORS` object. This makes it easier to update or rename classes. e.g., 
```js
const SELECTORS = {
  statContainer: ".impact-stat__number",
  digit: ".impact-stat__digit"
};
```